system_types.go

```diff
@@ -0,0 +1,42 @@
+package harbor
+
+// To ensure type-safe queries to the harbor API,
+// the following typings include typings from the upstream sources:
+// https://github.com/goharbor/harbor/src/core/api/models
+
+const (
+	// ScheduleHourly : 'Hourly'
+	ScheduleHourly = "Hourly"
+	// ScheduleDaily : 'Daily'
+	ScheduleDaily = "Daily"
+	// ScheduleWeekly : 'Weekly'
+	ScheduleWeekly = "Weekly"
+	// ScheduleCustom : 'Custom'
+	ScheduleCustom = "Custom"
+	// ScheduleManual : 'Manual'
+	ScheduleManual = "Manual"
+	// ScheduleNone : 'None'
+	ScheduleNone = "None"
+)
```

Comment:
maybe use an own type for the consts?

```suggestion
type ScheduleType string
const (
	ScheduleTypeHourly ScheduleType = "Hourly"
	ScheduleTypeDaily               = "Daily"
	ScheduleTypeWeekly              = "Weekly"
	ScheduleTypeCustom              = "Custom"
	ScheduleTypeManual              = "Manual"
	ScheduleTypeNone                = "None"
)
```